dags/elsevier/repository.py

```diff
@@ -0,0 +1,61 @@
+import os
+from io import BytesIO
+from typing import IO
+
+from common.repository import IRepository
+from common.s3_service import S3Service
+from common.utils import find_extension
+
+
+class ElsevierRepository(IRepository):
+    ZIPED_DIR: str = "raw/"
+    EXTRACTED_DIR: str = "extracted/"
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.s3 = S3Service(os.getenv("ELSEVIER_BUCKET_NAME", "elsevier"))
+
+    def get_all_raw_filenames(self):
+        return [
+            f.key.replace("raw/", "")
```

Comment:
I think `lstrip` might be better in this case

Reply:
☝️ instead of replace, won't that work? you don't go through the whole string to find matching pattern ;)

Reply:
For some reason in cuts the first letter, if "raw" is not present:
<img width="272" alt="Screenshot 2023-10-09 at 08 18 14" src="https://github.com/cern-sis/workflows/assets/36658177/34063512-15cb-4c98-bd8c-5793242d6087">